feature-libs/order/assets/translations/en/order.i18n.ts

```diff
@@ -108,6 +108,13 @@ export const order = {
         'Replenishment order #{{replenishmentOrderCode}} has been successfully cancelled',
     },
     caption: 'Order contents.',
+    deliveryPointOfServiceDetails: {
+      itemsToBePickUp: 'Items To Be Pick-Up',
+      pickUpInStoreAddress: 'Pick Up Store Address',
+      itemsToBeShipped: 'Items To Be Shipped',
+      shippingAddress: 'Shipping Address',
+      deliveryMethod: 'Delivery Method',
```

Comment:
shipping and deliveryMethod should no in `deliveryPointOfServiceDetails`, right? and maybe move this to bopis lib.

Reply:
Done